buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -0,0 +1,96 @@
+/** This file contains versions of all the dependencies used in the module  */
+
+const val kotlinVersion = "1.3.71"
```

Comment:
When you're using this kt version then why encouraged me to downgrade/revert it?

Reply:
my personal project. this is just about exploring. once I know all the new features then I can guide someone. 

I said to revert only if you don't know what went into 1.3.7 because I don't know yet